### DISCUSSION TITLE
v2.0.8

### DISCUSSION THREAD

**Discussion by alexdelprete** [alexdelprete/ha-abb-powerone-pvi-sunspec#28] [Announcements]
- Fixed bug in new model detection

<hr /><em>This discussion was created from the release <a href='https://github.com/alexdelprete/ha-abb-powerone-pvi-sunspec/releases/tag/v2.0.8'>v2.0.8</a>.</em>

**Comment by alexdelprete** [comment]
@dexmorgan8888 Just released 2.0.8. If you could test it, and also provide the value you see in the log error you'll find, because probably your model is not in the list yet. This won't break the integration, it will just throw an error because it can't find the model in the embedded list. I can add it based on the reported value.

**Reply by dexmorgan8888** [reply]
Sure, will do.
In the meantime, how to get the log?  Is this some special log?

**Reply by alexdelprete** [reply]
the HA log. :)

**Comment by dexmorgan8888** [comment]
2.0.8 seems to work... can't really tell cos it's night here, zero energy produced. Will have to see tomorrow.

The integration looks fine, I got 1 device with 30 entities. Previously with 2.0.7, there was nothing.
But still, there is an error in the log:

```
This error originated from a custom integration.

Logger: custom_components.abb_powerone_pvi_sunspec
Source: custom_components/abb_powerone_pvi_sunspec/__init__.py:193
Integration: ABB Power-One PVI SunSpec (documentation, issues)
First occurred: 22:22:26 (1 occurrences)
Last logged: 22:22:26

(opt_comm_model) Model unknown, report to @alexdelprete on the forum the following data: Manuf.: FIMER - Model: PVS-12.5-TL - Options: SINGAPORE - OptModel: SINGAPORE - OptModelInt: 83
```

**Reply by alexdelprete** [reply]
> The integration looks fine, I got 1 device with 30 entities. Previously with 2.0.7, there was nothing.
> But still, there is an error in the log:

It's not an error, like I explained in my previous post: if a model is not known (I have an internal list), I raise an error asking the user to contact me and provide the logging info. :)

Now I will add your model (PVS-12.5-TL) that is identified by code 83. When I release the new version (2.0.9), you will see the real model, instead of those strange 0x1234 strings.

I'll release it in a few minutes, so tomorrow you can try and tell me if it works (show me a screenshot of all the sensors in the dashboard).

Thanks for the help.
